packages/backend-function/src/factory.ts

```diff
@@ -121,6 +123,18 @@ export type FunctionProps = {
    * schedule: "0 9 * * 2" // every Monday at 9am
    */
   schedule?: FunctionSchedule | FunctionSchedule[];
+
+  /**
+   * Attach Lambda layers to a function
+   * @see [Lambda layers](https://docs.aws.amazon.com/lambda/latest/dg/chapter-layers.html)
+   * @example
+   * layers: {
+   *    "@aws-lambda-powertools/logger": "arn:aws:lambda:<current-region>:094274105915:layer:AWSLambdaPowertoolsTypeScriptV2:11"
+   * },
+   * @description The object is keyed by the module name hosted on your existing layer and a value that references to an existing layer using an ARN. The keys will be externalized and available via your layer at runtime
+   * @description Maximum of 5 layers can be attached to a function and must be in the same region as the function.
```

Comment:
`@description` is typically used to describe a symbol/type, is there another we could use? 

https://jsdoc.app/
https://tsdoc.org/

Reply:
switched to bullet points as the only other tags i could use are 
[@remarks](https://tsdoc.org/pages/tags/remarks/)
[@summary](https://jsdoc.app/tags-summary)